### PR TITLE
Add docstrings to EF

### PR DESCRIPTION
**Pull request by santiagoinfantinom**
Adresses #110

### CODE REVIEW

**Review by santiagoinfantinom — COMMENTED** [review]
@smartie2076 This is now ready for your review :)

File: src/F_case_definitions.py [hunk 1]
```diff
@@ -265,6 +284,32 @@ def update_dict(capacities_oem, specific_case, experiment):
...
+        Value for capacities or reference to them
+
+    capacities: dict
+        Contains capacities in the system
+
+    component_name: str
+        Name of the component from which its capacity will be extracted
+
+    batch_size: float
+        (?)
```
Comment (smartie2076) [inline src/F_case_definitions.py:305]:
```suggestion
        Undividable units that an asset can be installed as
```

File: src/F_case_definitions.py [hunk 2]
```diff
@@ -287,7 +332,22 @@ def get_base_capacity(
 
 
 def define_capacity(experiment_case_dict, case_dict_capacity, oemof_name):
+    """
+
```
Comment (smartie2076) [inline src/F_case_definitions.py:336]:
```suggestion
Updates the simulation information of a scenario with the capacities that each asset should already have pre-installed

Depends on whether assets in the scenario are to be optimized or if the capacity is determined based on a previous, but different scenario.
```

**Review by smartie2076 — APPROVED** [review]
More details to the functions can be added when generating the pytests :)

**Pull request merged**